assets/course-theme/blocks/lesson-blocks/index.js

```diff
@@ -259,4 +260,39 @@ export default [
 			);
 		},
 	},
+	{
+		...courseThemeLessonVideoMeta,
+		...meta,
+		title: __( 'Lesson Video (Learning Mode)', 'sensei-lms' ),
+		description: __(
+			'Displays the featured video if there is one for the lesson.',
+			'sensei-lms'
+		),
+		edit() {
+			return (
+				<div
+					className=""
+					style={ {
+						display: 'flex',
+						justifyContent: 'center',
+						alignItems: 'center',
+						height: '350px',
+					} }
+				>
+					<p
+						className="has-text-align-center"
+						style={ { fontSize: '100px' } }
+					>
+						<img
```

Comment:
I don't think we need to add this emoji image here, just the `▶` character as text should be good and even looks better.
(In the current cover placeholders WordPress actually replaces `▶` with the emoji image, it's not intentional :))

Also let's add a black background and white text for this block, so it approximates how a video player might look.